src/main/java/net/skinsrestorer/bukkit/SkinsRestorer.java

```diff
@@ -406,6 +407,9 @@ private void checkBungeeMode() {
             // we will try it again with the old function from SR 13.3
             if (!bungeeEnabled) {
                 bungeeEnabled = YamlConfiguration.loadConfiguration(new File("spigot.yml")).getBoolean("settings.bungeecord");
+                if (Files.exists(Paths.get("paper.yml"))) {
+                    bungeeEnabled = YamlConfiguration.loadConfiguration(new File("paper.yml")).getBoolean("settings.velocity-support.enabled");
```

Comment:
What if a server has:
* Paper
* Uses bungeecord
* And has velocity disabled 

Would break a lot of stuff @1MagicSweet

Reply:
Makes sense